src/models/user.js

```diff
@@ -19,13 +21,35 @@ const userSchema = new Schema({
   password: {
     type: String,
     required: true,
-    minLength: [6, "Password must be at least 6 characters long"],
+    minLength: [6, 'Password must be at least 6 characters long'],
+  },
+  confirmPassword: {
```

Comment:
`confirmPassword` should not be in the user schema. You only need it in the validator

Reply:
Alright, it'll be fixed on the next push. Question though, is it necessary to remove it if it won't be saved in the database. I used it only to confirm the password, but set it to undefined so it was never actually saved to the database